### PR TITLE
Add in pod convergence tests for classifier_trainner models.

### PR DESCRIPTION
**Pull request by allenwang28**
Should allow me to delete the BCT pod tests.

### CODE REVIEW

File: templates/garden/nightly/classifier-resnet.libsonnet [hunk 1]
```diff
@@ -73,11 +73,19 @@ local tpus = import "../../tpus.libsonnet";
...
+  local v3_32 = {
+    accelerator: tpus.v3_32,
+  },
 
   configs: [
     resnet + v2_8 + functional,
     resnet + v3_8 + functional,
     resnet + v2_8 + convergence,
     resnet + v3_8 + convergence,
+    resnet + v2_32 + convergence,
```
Comment (will-cromar) [inline templates/garden/nightly/classifier-resnet.libsonnet:88]: Please make these experimental for now. Automated pod tests are coming soon ™️

**Comment by allenwang28** [comment]
Synced with Will - marking this as draft until pod support is completed.

File: templates/garden/nightly/classifier-resnet.libsonnet (hunk 1 above)
Comment (allenwang28) [reply templates/garden/nightly/classifier-resnet.libsonnet:98]: Resolving this conversation as it should be obselete.

**Comment by allenwang28** [comment]
Made a few additional changes:
- Increased timeout for all classifier convergence tests
- Changed the regression metric to `val_epoch_accuracy_final` (this change was untested)

Functional test: https://pantheon.corp.google.com/kubernetes/job/europe-west4-a/europe-oneshots/default/tf-nightly-classifier-resnet-conv-v2-32-knzj9?project=xl-ml-test&tab=details&duration=P30D&pod_summary_list_tablesize=20&service_list_datatablesize=20

**Comment by allenwang28** [comment]
Sure I can, I did see an issue with the name becoming too long though. Should I change the name of the test?

**Comment by will-cromar** [comment]
I believe I shortened `functional` and `convergence` since you started this PR, which should buy you some extra characters. If the name is still too long, maybe you can shorten `classifier-efficientnet` to `efficientnet`?

**Comment by allenwang28** [comment]
You are correct, no issues this time. Added the functional tests

**Pull request merged**